### PR TITLE
NO JIRA. Amended config docs.

### PR DESCRIPTION
**Pull request by janvanmansum**
NO JIRA.

- [x] Question for @jo-pol  resolved.

#### When applied it will
* Update the style of the documentation comments in the config file.

### CODE REVIEW

File: src/main/assembly/dist/cfg/application.properties [hunk 1]
```diff
@@ -38,17 +38,27 @@ users.ldap-url=ldap://localhost
...
+# (Example: 3600 ( = 60 * 60 = is one hour ).
+#
 auth.cookie.expiresIn=3600
 
-# Pick a value starting with H, see fromString in https://github.com/pauldijou/jwt-scala/blob/master/core/common/src/main/scala/JwtAlgorithm.scala
+#
+# Algorithm and secret key to use for JSON Web Token. Algorithm can be any of the values supported by jwt-scala, see fromString in:
+#
+# https://github.com/pauldijou/jwt-scala/blob/master/core/common/src/main/scala/JwtAlgorithm.scala
+#
```
Comment (janvanmansum) [inline src/main/assembly/dist/cfg/application.properties]: @jo-pol : in the previous version of this comment it said a value starting with an H must be chosen? Why are the other values not valid?

Comment (jo-pol) [reply src/main/assembly/dist/cfg/application.properties]: Only hmac is imlpemented:
https://github.com/DANS-KNAW/easy-deposit-api/blob/5340ca35037c255aa76bdc7bff89113bb86ca711/src/main/scala/nl.knaw.dans.easy.deposit/authentication/TokenSupport.scala#L33-L44

Comment (jo-pol) [reply src/main/assembly/dist/cfg/application.properties]: The others a asymmetric algorithms and require two keys to encrypt/decrypt. We need only one key as cookies are both encrypted and decrypted on the server.

File: src/main/assembly/dist/cfg/application.properties [hunk 2]
```diff
@@ -38,17 +38,27 @@ users.ldap-url=ldap://localhost
...
-# In seconds, 60*60=3600 is one hour
+#
+# Lifetime for JSON Web Token cookie (named "scentry.auth.default.user") in seconds.
+# (Example: 3600 ( = 60 * 60 = is one hour ).
+#
 auth.cookie.expiresIn=3600
 
-# Pick a value starting with H, see fromString in https://github.com/pauldijou/jwt-scala/blob/master/core/common/src/main/scala/JwtAlgorithm.scala
+#
+# Algorithm and secret key to use for JSON Web Token. Algorithm can be any of the values supported by jwt-scala, see fromString in:
```
Comment (jo-pol) [inline src/main/assembly/dist/cfg/application.properties]:
```suggestion
# Symmetric algorithm and secret key to use for JSON Web Token. Algorithm can be any of the values implementing JwtHmacAlgorithm, see fromString in:
```

File: src/main/assembly/dist/cfg/application.properties (hunk 1 above)
Comment (janvanmansum) [reply src/main/assembly/dist/cfg/application.properties]: OK, thanks. I'll update the documentation comment. This way the future configurator (possibly you ;) will have a bit more understanding of the why of things.

**Pull request merged**